pdns/dnsbackend.cc

```diff
@@ -113,21 +113,16 @@ vector<string> BackendMakerClass::getModules()
 void BackendMakerClass::load_all()
 {
   // TODO: Implement this?
```

Comment:
Is this a leftover that can now be removed?

Reply:
I have no idea :-/ @Habbie, do you have any clue for us?

Reply:
There used to be `#ifndef WIN32` below this comment. My best guess is that the comment refers to implementing it for Windows.

Even if I'm wrong, clearly "this" is implemented, and the comment can go :-)

Reply:
Great, thanks a lot! I just pushed a commit removing the comment :)